src/or/channeltls.c

```diff
@@ -1583,6 +1583,18 @@ channel_tls_process_versions_cell(var_cell_t *cell, channel_tls_t *chan)
   }
 }
 
+/**
+ * Helper: compute the absolute value of a time_t.
+ *
+ * (we need this because labs() doesn't always work for time_t, since
+ * long can be shorter than time_t.)
+ */
+static inline time_t
+time_abs(time_t val)
+{
+  return (val < 0) ? -val : val;
```

Comment:
-TIME_T_MIN will overflow, but that's an issue in the existing code. (And very unlikely.)
Let's open a separate ticket in Tor: unspecified?

Reply:
Okay w me.